### PR TITLE
Making skimming articles work and Do Not Track support

### PR DESCRIPTION
**Pull request by Leiaz**
I unbroke article skimming, some changes to scroll functions had been lost. Apparently C programs don't mind much using the return value from a void function.

I also added the "do not track" setting support in the web extension.

### CODE REVIEW

File: src/ui/itemview.c [hunk 1]
```diff
@@ -360,13 +360,7 @@ itemview_find_unread_item (gulong startId)
 void
 itemview_scroll (void)
 {
-	/* We try to scroll the HTML view, but if we are already at the
-	   bottom of the item view the scrolling will return FALSE and 
-	   we trigger Next-Unread to realize easy headline skimming. */
-	if (liferea_htmlview_scroll (itemview->priv->htmlview) == FALSE)
-		on_next_unread_item_activate (NULL, NULL);
-		
-	/* Note the above condition is duplicated in mozilla/mozsupport.cpp! */
```
Comment (lwindolf) [inline src/ui/itemview.c]: LOL this comment was very old.

**Comment by Leiaz** [comment]
@lwindolf Do I merge it on master ?

**Comment by lwindolf** [comment]
@Leiaz Feel free if you are confident that it is stable. I sadly had no time to test yet :-(

**Comment by paulgevers** [comment]
I confirm that the first commit enables article skimming again (which I reported broken in #406). I have applied it as a patch in Debian.

**Pull request merged**